apps/server/src/modules/tool/external-tool/external-tool.module.ts

```diff
@@ -19,7 +20,24 @@ import {
 } from './service';
 
 @Module({
-	imports: [CommonToolModule, ToolConfigModule, LoggerModule, OauthProviderServiceModule, EncryptionModule, HttpModule],
+	imports: [
+		CommonToolModule,
+		ToolConfigModule,
+		LoggerModule,
+		OauthProviderServiceModule,
+		EncryptionModule,
+		HttpModule,
+		PDFModule.register({
```

Comment:
you are using the PdfService in the uc so you should import the module in the api module